packages/ui/react-ui/src/hooks/useCrossMintStatus.ts

```diff
@@ -19,14 +19,24 @@ interface IProps {
     clientId: string;
 }
 
+const isValidUUID = (uuid: string) =>
+    /^[0-9a-f]{8}-[0-9a-f]{4}-[0-5][0-9a-f]{3}-[089ab][0-9a-f]{3}-[0-9a-f]{12}$/i.test(uuid);
+
 export default function useCrossMintStatus({ clientId }: IProps) {
     const [status, setStatus] = useState<OnboardingRequestStatusResponse>(
         OnboardingRequestStatusResponse.WAITING_SUBMISSION
     );
 
     async function fetchClientIntegration() {
         if (!clientId || clientId === "" || clientId === "<YOUR_CLIENT_ID>") {
-            console.warn("You must enter your own CrossMint client ID in <CrossMintProvider clientId=XXX>");
+            console.error("You must enter your own Crossmint client ID in <CrossMintButton clientId=XXX>");
+            return;
+        }
+
+        if (!isValidUUID(clientId)) {
+            console.error(
+                "Entered `clientId` is not valid. It must be a valid Crossmint client ID. It will have the following format: `00000000-0000-0000-0000-00000000`."
```

Comment:
Thoughts on the error message? Do you think we should be even more explicit? Maybe with a code snippet of the button?

Reply:
For now I'd say "The clientId passed to <CrossMintButton> is invalid. Make sure to pass the clientId obtained from the crossmint team, with format `XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX`"

I think we can improve this later to pointing to a URL within our documentation site that explains how client IDs work. But we don't have a great place yet